sklearn/preprocessing/data.py

```diff
@@ -1948,6 +1948,11 @@ class QuantileTransformer(BaseEstimator, TransformerMixin):
     subsample : int, optional (default=1e5)
         Maximum number of samples used to estimate the quantiles.
 
+    noise_variance : None or float, optional (default=None)
```

Comment:
`smoothing_noise`? Anyway this is not the variance but the standard deviation of the normal distribution.